packages/blade/src/components/Tooltip/Tooltip.web.tsx

```diff
@@ -103,6 +104,7 @@ const Tooltip = ({
             ref={refs.setFloating}
             style={floatingStyles}
             pointerEvents="none"
+            zIndex={zIndex}
```

Comment:
Added zIndex prop to Tooltip, because we don't know if consumer's wrapper will have 100 or 200 zIndex it can be larger. 

In that cases better to let consumers customise it.

Reply:
Removed and assigned a higher value. 

Raised an issue to track the refactor: https://github.com/razorpay/blade/issues/1402